src/arb/arb.jl

```diff
@@ -1653,14 +1653,14 @@ function sinhcosh(x::arb)
 end
 
 @doc Markdown.doc"""
-    atan2(x::arb, y::arb)
+    atan2(y::arb, x::arb)
```

Comment:
I don't understand why you switched x and y.

Reply:
I guess because x + yi looks more natural?

Reply:
if the point in the complex plane is `x+i*y` the standard argument is obtained via `atan2(y,x)`.
It matches
https://docs.julialang.org/en/v1/base/math/#Base.atan-Tuple{Number}